dataflow/run-inference/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM apache/beam_python3.11_sdk:2.48.0
+
+COPY requirements.txt ./
+COPY *.py ./
+
+RUN pip install --no-cache-dir -U pip \
+    && pip install --no-cache-dir -r requirements.txt \
+    && pip check
```

Comment:
I believe you need add:
```
# Set the entrypoint to Apache Beam SDK launcher.
ENTRYPOINT ["/opt/apache/beam/boot"]
```
for custom container to work.

Reply:
Tests are passing like this, I suspect the base image might already have defined it and this one is inheriting it?

Reply:
Oh, I agree with you.